bats/tests/containers/factory-reset.bats

```diff
@@ -86,16 +86,30 @@ check_installation() {
         # this one only exists after an update has been downloaded
         # ~/Library/Application Support/Caches/rancher-desktop-updater
     fi
+
+    if is_windows; then
+        delete_dir+=("$PATH_LOGS" "$PATH_DISTRO" "$PATH_DISTRO_DATA")
+    fi
+
     for dir in "${delete_dir[@]}"; do
         echo "$assert that $dir does not exist"
         ${assert}_not_exists "$dir"
     done
 
+    #Check if rancher-desktop WSL distros are deleted on Windows
+    if is_windows; then
+        run wsl --list
+        ${refute}_output "rancher-desktop"
+        ${refute}_output "rancher-desktop-data"
```

Comment:
These can't be true at the same time (and also ignore that there is a header line.

```suggestion
        ${refute}_line "rancher-desktop"
        ${refute}_line "rancher-desktop-data"
```